lib/kb/models/hubspot.rb

```diff
@@ -0,0 +1,34 @@
+module KB
+  class Hubspot < BaseModel
```

Comment:
This class looks like it's already representing a relationship so the name should reflect that:

```suggestion
  class HubspotRelationship < BaseModel
```